src/app/request-handlers/guards/start-date-guard.ts

```diff
@@ -7,7 +7,8 @@ export class StartDateGuard extends RequestHandlerGuard {
     }
 
     public async check(onReject: (message: string) => Promise<void>): Promise<void> {
+        // TODO: Update reject message
         if (compareAsc(this.submittedDate, this.startDate) == -1)
-            onReject('Request was submitted before the start date');
+            onReject('Request was submitted before the available from date/time');
```

Comment:
Similar to the end date guard, can we just say "Request is submitted too early"?